lib/dsl/chev_grammar.js

```diff
@@ -91,12 +102,18 @@ createToken({ name: 'ANGULAR_SUFFIX', pattern: 'angularSuffix' });
 
 // validations
 createToken({ name: 'REQUIRED', pattern: 'required' });
-createToken({ name: 'MINLENGTH', pattern: 'minlength' });
-createToken({ name: 'MAXLENGTH', pattern: 'maxlength' });
-createToken({ name: 'MINBYTES', pattern: 'minbytes' });
-createToken({ name: 'MAXBYTES', pattern: 'maxbytes' });
-createToken({ name: 'MAX', pattern: 'max' });
-createToken({ name: 'MIN', pattern: 'min' });
+
+// HIGHLIGHT:
+// "MIN_MAX_KEYWORD" is an "abstract" token which other concrete tokens inherit from.
+// This can be used to reduce verbosity in the parser.
```

Comment:
How so?

Reply:
It is used in the validation rule instead of specifying the six different keywords.
https://github.com/jhipster/jhipster-core/pull/142/files#diff-802ee05eaf770a8bbbc2fe7ef13a3efaR233

Reply:
Here is the corresponding section in the existing grammar:
https://github.com/jhipster/jhipster-core/blob/fd8f71200abcb4ffccb7063a4312641289c090c4/lib/dsl/grammar.txt#L511-L522

The token inheritance does not have to be used.
It is an example for what is possible and could be considered...